### PR TITLE
[Doc] Add Chinese translation of install.md and 4 super-resolution datasets README.md

### PR DESCRIPTION
**Pull request by LiUzHiAn**
This PR adds a Chinese version of the installation instructions (`install.md`) and the 4 super-resolution datasets description files (in `tools/data/super-resolution/*/README.md`).

### CODE REVIEW

**Review by ckkelvinchan — COMMENTED** [review]
Some comments. Thank you very much for your help.

File: docs_zh-CN/install.md [hunk 1]
```diff
@@ -1,11 +1,96 @@
 ## 依赖
 
-- Linux (Windows is not officially supported)
+- Linux (目前Windows不是官方支持)
```
[the remark this answers is not in the document]
Comment (LiUzHiAn) [reply docs_zh-CN/install.md:3]: Got it, will fix them later.

File: docs_zh-CN/install.md [hunk 2]
```diff
@@ -1,11 +1,96 @@
...
 - [mmcv](https://github.com/open-mmlab/mmcv)
 
 ## 安装
+a. 创建并激活 conda 虚拟环境，如：
+
+```shell script
+conda create -n open-mmlab python=3.7 -y
+conda activate open-mmlab
+```
+b. 按照 [PyTorch官方文档](https://pytorch.org/) 安装 PyTorch 和 torchvision，如：
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:18]: PyTorch官方文档 -> PyTorch 官方文档

File: docs_zh-CN/install.md [hunk 3]
```diff
@@ -1,11 +1,96 @@
...
+
+```shell script
+conda install pytorch torchvision -c pytorch
+```
+注意：确保你的 CUDA 编译版本和 CUDA 运行版本相匹配。 用户可以参照 [PyTorch官网](https://pytorch.org/) 对预编译包所支持的 CUDA 版本进行核对。
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:23]: PyTorch官网 -> PyTorch 官网

File: docs_zh-CN/install.md [hunk 4]
```diff
@@ -1,11 +1,96 @@
...
+```shell script
+pip uninstall mmedit
+rm -rf ./build
+find . -name "*.so" | xargs rm
+```
+2. 根据上述步骤， MMEditing 就会以 `dev` 模式被安装，任何本地的代码修改都会立刻生效，不需要再重新安装一遍（除非用户提交了 commits，并且想更新版本号）。
+
+3. 如果用户想使用 `opencv-python-headless` 而不是 `opencv-python`，可在安装 `MMCV` 前安装 `opencv-python-headless`。
+
+4. 有些模型（例如图像修复任务中的 `EDVR`）依赖于`mmcv-full`中的一些 CUDA 算子，具体的依赖可在`requirements.txt`中找到。
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:62]: 依赖于`mmcv-full`中 -> 依赖于 `mmcv-full` 中

在`requirements.txt`中 -> 在 `requirements.txt` 中

File: docs_zh-CN/install.md [hunk 5]
```diff
@@ -1,11 +1,96 @@
...
+如需要，请通过 `pip install -r requirements.txt`命令来安装`mmcv-full`，安装过程中会在本地编译 CUDA 算子，这个过程大概需要10分钟。
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:63]: 请通过 `pip install -r requirements.txt`命令 -> 请通过 `pip install -r requirements.txt` 命令

File: docs_zh-CN/install.md [hunk 6]
```diff
@@ -1,11 +1,96 @@
...
+另一种方案是安装预编译版本的`mmcv-full`，请参考 [MMCV主页](https://github.com/open-mmlab/mmcv#install-with-pip) 获取具体的安装指令。
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:64]: 版本的`mmcv-full` -> 版本的 `mmcv-full`

MMCV主页 -> MMCV 主页

File: docs_zh-CN/install.md [hunk 7]
```diff
@@ -1,11 +1,96 @@
...
+此外，如果你要使用的模型不依赖于 CUDA 算子，那么也可以使用`pip install mmcv`来安装轻量版本的 mmcv，其中 CUDA 算子被移除了。
```
Comment (ckkelvinchan) [inline docs_zh-CN/install.md:65]: 使用`pip install mmcv`来 -> 使用 `pip install mmcv` 来

File: tools/data/super-resolution/README_zh-CN.md [hunk 8]
```diff
@@ -1 +1,9 @@
-# 数据集 - 复原
+# 超分辨数据集
+建议将数据集的根目录链接到`$MMEDITING/data`下，如果你的文件目录结构不一致，那么你可能需要在配置文件中修改对应的文件路径。
```
Comment (ckkelvinchan) [inline tools/data/super-resolution/README_zh-CN.md:2]: 到`$MMEDITING/data`下 -> 到 `$MMEDITING/data` 下

File: tools/data/super-resolution/reds/README_zh-CN.md [hunk 9]
```diff
@@ -11,3 +11,44 @@
   year = {2019}
 }
 ```
+
+- 训练集: [REDS 数据集](https://seungjunnah.github.io/Datasets/reds.html).
+- 验证集: [REDS 数据集](https://seungjunnah.github.io/Datasets/reds.html) 和 Vid4.
+
+请注意，我们合并了 REDS 的训练集和验证集，以便在 REDS4 划分（在`EDVR`中会使用到）和官方验证集划分之间切换。
```
Comment (ckkelvinchan) [inline tools/data/super-resolution/reds/README_zh-CN.md:18]: 在`EDVR`中 -> 在 `EDVR` 中

File: tools/data/super-resolution/reds/README_zh-CN.md [hunk 10]
```diff
@@ -11,3 +11,44 @@
...
+
+原始验证集的名称被修改了（clip 000 到 029），以避免与训练集发生冲突（总共 240 个clip）。具体而言，验证集中的 clips 被改名为 240、241、... 269。
```
Comment (ckkelvinchan) [inline tools/data/super-resolution/reds/README_zh-CN.md:20]: 总共 240 个clip -> 总共240个 clip

File: tools/data/super-resolution/vimeo90k/README_zh-CN.md [hunk 11]
```diff
@@ -14,3 +14,36 @@
   publisher={Springer}
 }
 ```
+训练集和测试集可以从 [此处](http://toflow.csail.mit.edu/) 下载。
+
+Vimeo90K 数据集包含了如下所示的`clip/sequence/img`目录结构：
```
Comment (ckkelvinchan) [inline tools/data/super-resolution/vimeo90k/README_zh-CN.md:19]: 所示的`clip/sequence/img`目录 -> 所示的 `clip/sequence/img` 目录

**Review by ckkelvinchan — APPROVED** [review]

**Comment by ckkelvinchan** [comment]
@LiUzHiAn Please install [pre-commit](https://pre-commit.com/). This helps you check the format of the modified files.

After installing pre-commit, you can `cd mmediting` and enter `pre-commit install`.

**Comment by LiUzHiAn** [comment]
> @LiUzHiAn Please install [pre-commit](https://pre-commit.com/). This helps you check the format of the modified files.
> 
> After installing pre-commit, you can `cd mmediting` and enter `pre-commit install`.

After installing `pre-commit`, do I need to make a dummy commit?

**Comment by ckkelvinchan** [comment]
> > @LiUzHiAn Please install [pre-commit](https://pre-commit.com/). This helps you check the format of the modified files.
> > After installing pre-commit, you can `cd mmediting` and enter `pre-commit install`.
> 
> After executing `pre-commit run --all-files`, the yapf related error was raised, just something as :
> 
> ```
> yapf.....................................................................Failed
> - hook id: yapf
> - exit code: 1
> ...
> ...
>  File "/home/liuzhian/anaconda3/envs/mmlab/lib/python3.6/lib2to3/pgen2/parse.py", line 159, in addtoken
>     raise ParseError("bad input", type, value, context)
> lib2to3.pgen2.parse.ParseError: bad input: type=3, value="'\\btitle\\s*=\\s*{{\\s*{p}\\s*}}.*?\\n## (.*?)\\s*[,;]?\\s*\\n'", context=('', (44, 19))
> ```
> 
> Any idea to fix that? Thanks for your time

Usually, I commit the changes as usual. If `pre-commit` is installed, it will parse your files and change some minor formatting errors for you. You can try that.

**Comment by LiUzHiAn** [comment]
> > > @LiUzHiAn Please install [pre-commit](https://pre-commit.com/). This helps you check the format of the modified files.
> > > After installing pre-commit, you can `cd mmediting` and enter `pre-commit install`.
> > 
> > 
> > After executing `pre-commit run --all-files`, the yapf related error was raised, just something as :
> > ```
> > yapf.....................................................................Failed
> > - hook id: yapf
> > - exit code: 1
> > ...
> > ...
> >  File "/home/liuzhian/anaconda3/envs/mmlab/lib/python3.6/lib2to3/pgen2/parse.py", line 159, in addtoken
> >     raise ParseError("bad input", type, value, context)
> > lib2to3.pgen2.parse.ParseError: bad input: type=3, value="'\\btitle\\s*=\\s*{{\\s*{p}\\s*}}.*?\\n## (.*?)\\s*[,;]?\\s*\\n'", context=('', (44, 19))
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Any idea to fix that? Thanks for your time
> 
> Usually, I commit the changes as usual. If `pre-commit` is installed, it will parse your files and change some minor formatting errors for you. You can try that.

I seem to find the reason. It's because some lines of code in `mmedit/models/backbones/encoder_decoders/decoders/fba_decoder.py` don't meet the yapf format, but I never modify. Just running `pre-commit run` will force the `pre-commit` to skip this issue.

Indeed, commit the changes as usual will also lead to the `pre-commit run` command. Anyway, I've committed my final Docs change.

**Comment by ckkelvinchan** [comment]
> > > > @LiUzHiAn Please install [pre-commit](https://pre-commit.com/). This helps you check the format of the modified files.
> > > > After installing pre-commit, you can `cd mmediting` and enter `pre-commit install`.
> > > 
> > > 
> > > After executing `pre-commit run --all-files`, the yapf related error was raised, just something as :
> > > ```
> > > yapf.....................................................................Failed
> > > - hook id: yapf
> > > - exit code: 1
> > > ...
> > > ...
> > >  File "/home/liuzhian/anaconda3/envs/mmlab/lib/python3.6/lib2to3/pgen2/parse.py", line 159, in addtoken
> > >     raise ParseError("bad input", type, value, context)
> > > lib2to3.pgen2.parse.ParseError: bad input: type=3, value="'\\btitle\\s*=\\s*{{\\s*{p}\\s*}}.*?\\n## (.*?)\\s*[,;]?\\s*\\n'", context=('', (44, 19))
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Any idea to fix that? Thanks for your time
> > 
> > 
> > Usually, I commit the changes as usual. If `pre-commit` is installed, it will parse your files and change some minor formatting errors for you. You can try that.
> 
> I seem to find the reason. It's because some lines of code in `mmedit/models/backbones/encoder_decoders/decoders/fba_decoder.py` don't meet the yapf format, but I never modify. Just running `pre-commit run` will force the `pre-commit` to skip this issue.
> 
> Indeed, commit the changes as usual will also lead to the `pre-commit run` command. Anyway, I've committed my final Doc changes.

It worked now, thanks!

File: tools/data/super-resolution/README_zh-CN.md [hunk 12]
```diff
@@ -1 +1,9 @@
-# 数据集 - 复原
+# 超分辨数据集
```
Comment (innerlee) [inline tools/data/super-resolution/README_zh-CN.md:1]: 超分辨 -> 超分辨率

Comment (innerlee) [reply tools/data/super-resolution/README_zh-CN.md:1]: see https://github.com/open-mmlab/mmediting/issues/432

File: docs_zh-CN/install.md [hunk 13]
```diff
@@ -1,11 +1,96 @@
...
+# build an image with PyTorch 1.5, CUDA 10.1
+docker build -t mmediting docker/
+```
+运行以下命令：
+```shell script
+docker run --gpus all --shm-size=8g -it -v {DATA_DIR}:/mmediting/data mmediting
+```
+
+## 完整的安装脚本
+```shell script
```
Comment (innerlee) [inline docs_zh-CN/install.md:86]: add an empty line between header and contents

File: tools/data/super-resolution/div2k/README_zh-CN.md [hunk 14]
```diff
@@ -11,3 +11,81 @@
...
+│   │   │   ├── X4_sub
+│   │   ├── DIV2K_valid_HR
+│   │   ├── ...
+...
+```
+
+## 准备标注列表文件
+如果你想使用`标注模式`来处理数据集，你需要先准备一个 `txt` 格式的标注文件。
+
+标准文件中的每一行包含了图片名以及图片尺寸（这些通常是 ground-truth 图片），这两个字段用空格间隔开。
```
Comment (innerlee) [inline tools/data/super-resolution/div2k/README_zh-CN.md:76]: 标准文件

**Comment by OpenMMLab-Coodinator** [comment]
Hi @LiUzHiAn ！First of all, we want to express our gratitude for your significant PR in this project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**